packages/@expo/config/src/evalConfig.ts

```diff
@@ -102,6 +106,13 @@ export function resolveConfigExport(
   configFile: string,
   request: ConfigContext | null
 ) {
+  // add key to static config that we'll check for after the dynamic is evaluated
+  // to see if the static config was used in determining the dynamic
+  const hasBaseStaticConfig = Symbol();
```

Comment:
Best practice is usually to move this out to the global and give it a name. ref https://github.com/expo/expo/blob/923016204c239cdad07b89626ac65eb505623d98/packages/%40expo/server/src/environment.ts#L50